jaxrs-tck/pom.xml

```diff
@@ -78,6 +80,18 @@
         <tag>HEAD</tag>
     </scm>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>5.7.0</version>
```

Comment:
Do you deliberately have different version numbers in lines 35 and 88?

Reply:
No, I copied that from other TCK examples.  I think we'll probably want to use the latest level, but we've got to get it working first. :)